default-fields/0000-default-fields.md

```diff
@@ -6,31 +6,268 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow structure definitions to supply default values for individual fields, and then allow those fields to be omitted from struct initialisation:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32,
+}
+
+let foo = Foo {
+    a: "Hello",
+    c: 42,
+};
+```
 
 # Motivation
 [motivation]: #motivation
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Field defaults address two issues with structure initialisation in Rust: privacy and boilerplate. This is achieved by letting callers omit fields from initialisation when a default is specified for that field. This syntax also allows allows fields to be added to structures in a backwards compatible way, by providing defaults for new fields.
+
+Rust allows you to create an instance of a structure using a literal syntax. This requires all fields in the structure be assigned a value, and can't be used with private fields. It can also be inconvenient for large structures whose fields usually receive the same values.
+
+With the `..` syntax, values for missing fields can be taken from another structure. However, this still requires an already initialised struct after the `..`. It also isn't valid if the struct has innaccessible private fields.
```

Comment:
@KodrAus The private field trick would be a hypothetical trick if we used explicit syntax for default values. Currently it dosent work because FRU and private fields dont go along.